src/main/kotlin/com/openlattice/shuttle/Shuttle.kt

```diff
@@ -412,10 +481,8 @@ class Shuttle(
                                     .add(Association(key, src, dst, data))
                         }
                     } else {
-                        logger.error(
-                                "Encountered blank entity id for entity set {}",
-                                associationDefinition.entitySetName
-                        )
+                        val blankEntityIdError = "Encountered blank entity id for entity set ${associationDefinition.entitySetName}"
+                        writeLog(flight.name, blankEntityIdError, OffsetDateTime.now(), IntegrationStatus.IN_PROGRESS)
```

Comment:
Does any call to writeLog pass something other than OffsetDateTime.now() ? if not, can we remove that parameter? (Or make it the 4th parameter with a default value of OffsetDateTime.now() so that it doesn't need to be passed the bulk of the time?)